docs/contribute/whiteboard.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Saros HTML Whiteboard
+---
+
+# {{ page.title }}
+
+The Saros Whiteboard was designed to enable Saros users to communicate ideas graphically.
+
+## Developing the Whiteboard
+
+### Setup
+Use the command `npm run setup`, note that `npm install` does not suffice.
+
+### Build for Development
+Use the command `npm run dev` to run webpack in watch mode, any changes to the code will automatically trigger the bundling. To test the whiteboard, simply open the `index.html`. You can adjust the build configurations in the `webpack.config.js` file.
+
+### Build for Production
+Use the command `npm run build`. When building for production, all of the resources of the whiteboard will be bundled into one file `dist/index.html`.
+
+## Technology Stack
+The Saros HTML Whiteboard uses a graphics library called [Fabric.js](http://fabricjs.com/) to display the shapes in the page, it also uses [React](https://reactjs.org/) to render the menu.
+
+As for the bundling, it uses [Webpack](https://webpack.js.org/) to bundle the resources using [Babel](https://babeljs.io/) loader to transpile the code to an older more-browser-friendly version.
+
+## How it works internally
+
+The Saros HTML Whiteboard extends the [GEF (Graphical Editing Framework)](https://www.eclipse.org/gef/) based Whiteboard and uses most of the internal structure such as the [SXE](https://xmpp.org/extensions/xep-0284.html).
```

Comment:
Does it really extends something ? Isn't the HTML Whiteboard a REPLACEMENT for the currently uses Whiteboard which is only available when using Eclipse because it uses the GEF ?

Reply:
Agree, "extends" does not fit the HTML Whiteboard, I have changed it.